src/main/java/no/nav/familie/integrasjoner/client/rest/SafRestClient.kt

```diff
@@ -71,6 +77,15 @@ class SafRestClient(@Value("\${SAF_URL}") safBaseUrl: URI,
         }
     }
 
+    fun hentDokument(journalpostId: String, dokumentInfoId: String, variantFormat: String): ByteArray {
```

Comment:
Det er ju fint å ha de i den samme, men det blir også litt forskjell. Her er det nå mulig å hente dokumentet hvis det er systemcontext.
Si eks att vi i ef-sak gjør feil og kaller hentDokument med systemcontext (eks att vi bruker feil interceptor og ikke sender obo-token), då får vi hentet dokumentet, då vi kaller tjenesten med sts, men burde kanskje ikke fått gjøre det? 

Så fordelen med å ha denne i en egen restclient med explicit obo er att vi garanterar att vi kaller på hentDokument med obo.

Reply:
Argumentet gjelder vel mot andre integrasjoner også - for eksempel pdl? Hva er logikken der?

Reply:
Jo det er sant. Det er litt av ulempen når man har ett api som tillåter både system og obo tokens. Då blir det konsumentene som blir ansvarlige for att de får hente gitt data. 
Ulempen då er att hvis man ikke har konfigurert appen riktig med client_registrations, og kaller på integrasjoner med sts/client_ så får man hentet data som man egentlige ikke har tilgang til.

Reply:
Yes, når det gjelder PDL så har vi forsåvidt flyttet det meste av logikken til saksapplikasjonene... Kanskje vi kan beholde hentDokument på kun obo 👍